text/0021-versioning-releases.md

```diff
@@ -0,0 +1,26 @@
+# Releasing and Versioning of Openmrs SingleSpa Apps
+- Start Date: 2020/01/21
+- RFC PR: https://github.com/openmrs/openmrs-rfc-frontend/pull/1
+
+## Decision
+Openmrs spa applications should have releases that are versioned so that each implementor can have control on when to upgrade to the latest changes. 
```

Comment:
"SPA applications" is redundant. "can have control on" -> "has control over". "when" -> "when and whether". "the latest changes" -> "more recent versions".